README.md

```diff
@@ -126,10 +126,11 @@ Place a breakpoint at the line `NSLog(@"Hello Objective-C");` and run from Visua
 
 ## Status and Known Issues
 
-The toolchain is fully usable on x64. On x86, libdispatch is not available due to a [build error](https://bugs.swift.org/browse/SR-14314).
+- The toolchain is fully usable on x64. On x86, libdispatch is not available due to a [build error](https://bugs.swift.org/browse/SR-14314).
```

Comment:
Nit: let’s use asterisks for this list to match the lists below.
```suggestion
* The toolchain is fully usable on x64. On x86, libdispatch is not available due to a [build error](https://bugs.swift.org/browse/SR-14314).
```

Reply:
> Nit: let’s use asterisks for this list to match the lists below.

Ofc